locobot/robot/remote_locobot.py

```diff
@@ -184,12 +179,10 @@ def go_to_relative(self, xyt_position, use_map=False, close_loop=True, smooth=Fa
         :type close_loop: bool
         :type smooth: bool
         """
-        if self._done:
-            self._done = False
-            self._robot.base.go_to_relative(
-                xyt_position, use_map=use_map, close_loop=close_loop, smooth=smooth
-            )
-            self._done = True
+        if self._move_done:
+            self._move_done = False
+            self._slam.set_relative_goal_in_pr_frame(xyt_position)
```

Comment:
let's still keep the pyrobot APIs around for now, for redundancy. I'd add another env variable (called `use_dslam`) and pass it into the LoCoBotMover constructor and use that to decide whether I should call the slam api or the pyrobot api.

Reply:
added `use_dslam` to switch between pyrobot api and slam api